components/infobox/extensions/commons/years_active_base.lua

```diff
@@ -62,6 +62,7 @@ function ActiveYears.display(args)
 		error('"playerPositionLimit" has to be >= 1')
 	end
 
+	-- Build conditions
```

Comment:
Don't see how the changes in this module are related to the PR (they don't actually change anything)

Reply:
The `_calculate` function was split into two so I could make use of just one part of the function

Reply:
Ok